CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change history for ui-marc-authorities
 
+## [6.1.0] (IN PROGRESS)
+
+- [UIMARCAUTH-445](https://issues.folio.org/browse/UIMARCAUTH-445) MARC authority > View Source > Display Version History pane with an empty Version History component.
```

Comment:
Shouldn't it be a second pane?

Reply:
It will be a second pane for View Source for Inventory records because there View Source is displayed in full screen.
I'll ask Christine to update the story